lib/roo_on_rails/railties/http.rb

```diff
@@ -28,10 +28,13 @@ class HTTP < Rails::Railtie
           app.config.middleware.use ::Rack::Deflater
         end
 
-        app.config.middleware.insert_before(
-          ActionDispatch::Cookies,
-          ::Rack::SslEnforcer
-        )
+        # Don't use SslEnforcer in test environment as it breaks Capybara
+        unless Rails.env.test?
+          app.config.middleware.insert_before(
+            ActionDispatch::Cookies,
```

Comment:
We can try, but the relative ordering of this to cookies is specifically important as it interacts with it differently depending on ordering. 

If you're running in API mode how are you going to secure the Sidekiq dashboard? Or will you not have one?